docs/source/specs/openapi.yml

```diff
@@ -0,0 +1,705 @@
+openapi: 3.0.0
```

Comment:
most of this page can be autogenerated. incase, you haven't any manual changes, maybe we can add the command in our CI, to keep this up to date?

Reply:
In fact I did the job manually, I faced some issue with autogenerate provided by the rest framework and the time to write this manually was faster than debugging the issue and I also wanted to make the open api for the comment app only while the framework will try to generate it for the test project as well.

Reply:
okay, then this can be addressed in a separate issue.

Reply:
I am not sure if that could be easily achieved, otherwise, it's not worth the time that will be spent on it.